Ajout de nouveaux fichiers assets.css et formations.css

w0wonder opened the pull request:
Il m'a fallu créer de nouveaux fichiers avec les mêmes contenus parce que sans cela, je ne parvenais pas à faire le lien entre les différents fichiers pour avoir le rendu escompté.

w0wonder committed:
Add files via upload

w0wonder committed:
formations.html

w0wonder committed:
Add files via upload

w0wonder committed:
Merge branch 'main' of https://github.com/w0wonder/SkillUp

w0wonder committed:
k

w0wonder committed:
Merge branch 'jossweb:main' into main

w0wonder committed:
Merge branch 'jossweb:main' into main

w0wonder committed:
Ajout du fichier  assets/css/example.css et modification de
formations.html

w0wonder committed:
Merge branch 'main' of https://github.com/w0wonder/SkillUp

w0wonder committed:
Modification du fichier assets/css/formations.css

w0wonder committed:
Merge branch 'jossweb:main' into main

w0wonder committed:
Create formations.php

w0wonder committed:
Add files via upload

w0wonder committed:
Modification de assets/css/example.css
Modification de assets/css.formations.css
Modification de formations.html

w0wonder committed:
Modification assets/css/formations.css
Modification formations.html

w0wonder committed:
Suppression assest/css/formations.css
Modification formations.html

w0wonder committed:
Ajout assets.css
Ajout formations.css

w0wonder committed:
Insertion assets.css formations.css et formations.html

w0wonder committed:
Nouveau fichier assets/css/formations.css
Nouveau fichier assets/html/assets.css
Nouveau fichier assets/html/formations.html
Nouveau fichier assets/html/assets.css

w0wonder committed:
Update formations.php

jossweb commented:
Cette proposition ne convient pas pour le moment.

Les images doivent être placées dans le dossier assets/images et les fichiers CSS dans assets/css. Par ailleurs, je ne comprends pas pourquoi une page HTML a été ajoutée dans le dossier assets, alors que toutes les pages doivent se trouver dans le répertoire /pages.

Concernant le fichier formation.php, il y a un souci : le fichier contient le code de deux pages, ce qui n’est pas conforme. Cela semble provenir d’un copier-coller un peu trop rapide !

En l’état, je ne peux donc pas accepter cette pull request. N’hésite pas à me contacter si tu as des questions ou besoin de précisions.

w0wonder committed:
Rename formations.html to formations.php

w0wonder committed:
suppression assets/html/assets.css
supppression assets/htmml/formations.css
suppression formations.php
renommé assets/html/formations.html en pages/formations.php

w0wonder committed:
Merge branch 'main' of https://github.com/w0wonder/SkillUp

w0wonder commented:
> Cette proposition ne convient pas pour le moment.
> 
> Les images doivent être placées dans le dossier assets/images et les fichiers CSS dans assets/css. Par ailleurs, je ne comprends pas pourquoi une page HTML a été ajoutée dans le dossier assets, alors que toutes les pages doivent se trouver dans le répertoire /pages.
> 
> Concernant le fichier formation.php, il y a un souci : le fichier contient le code de deux pages, ce qui n’est pas conforme. Cela semble provenir d’un copier-coller un peu trop rapide !
> 
> En l’état, je ne peux donc pas accepter cette pull request. N’hésite pas à me contacter si tu as des questions ou besoin de précisions.

Salut, j'ai apporté les modifications. J'avais un peu de mal avec les liens entre les fichiers et j'ai demandé à Tenin qui m'a guidée. Tu peux vérifier et me revenir. Merci
